Makefile

```
@@ -1,8 +1,24 @@
+# copy .env-example to .env and modify contents
+include .env
+
+BUILD_DIR := build/tmp/deploy/images/iot2000/
+BUILD_FILE := core-image-minimal-iot2000.wic
+BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+HASH := $(shell git rev-parse HEAD)
```

Keep file names short use short SHA
```suggestion
HASH := $(shell git rev-parse --short HEAD)
```